src/components/StatusBadge.jsx

```diff
@@ -13,17 +13,21 @@ export const StatusBadge = ({ className, status }) => {
     label: status,
     variant: 'light',
   };
-  if (status === statuses.ungraded) {
-    args = { label: 'Ungraded', variant: 'primary' };
-  }
-  if (status === statuses.locked) {
-    args = { label: 'Grading in progress', variant: 'light' };
-  }
-  if (status === statuses.graded) {
-    args = { label: 'Grading Complete', variant: 'success' };
-  }
-  if (status === statuses.inProgress) {
-    args = { label: 'Locked by you', variant: 'warning' };
+  switch(status) {
+    case statuses.ungraded:
+      args = { label: 'Ungraded', variant: 'primary' };
+      break;
+    case statuses.inProgress:
+      args = { label: 'Grading in progress', variant: 'light' };
+      break;
+    case statuses.graded:
+      args = { label: 'Grading Complete', variant: 'success' };
+      break;
+    case statuses.locked:
+      args = { label: 'Locked by you', variant: 'warning' };
+      break;
+    default:
+      break;
```

Comment:
why?

Reply:
Earlier it was wrong on in progress and locked. It was confusing when I did filtering. Since I edit it, might as well optimize it a little. I can change it to `else if` if that is prefer.

Reply:
If anything, I would set up a dict for this lookup
```
const statusArgs = {
  <statuses.ungraded>: { label: 'Ungraded', variant: 'primary' },
  ...
};
...
args = statusArgs[status]
```